www/base/src/app/settings/service/settings.service.coffee

```diff
@@ -0,0 +1,45 @@
+class bbSettings extends Provider
```

Comment:
the class is injected below as bbSettingsService. As I'm quite new to AngularJS, I'm curious to understand what's the magic that make the link between bbSettings and bbSettingsService